src/traccuracy/_tracking_graph.py

```diff
@@ -201,44 +201,22 @@ def __init__(
         self.node_errors = False
         self.edge_errors = False
 
-    def nodes(self, limit_to=None):
+    def nodes(self):
```

Comment:
Can we change `nodes` and `edges` to `@property` so that they aren't callable and match the networkx behavior? Of course at that point it's close to just justifying getting rid of them entirely but constantly having to refer to the graph attribute gets ugly.

Reply:
Yep, agreed @msschwartz21